app/views/layouts/_header.html.erb

```diff
@@ -6,7 +6,7 @@
             <li><%= link_to "About", about_path %></li>
             <li><%= link_to "Contact Us", contact_path %></li>
             <% if logged_in? %>
-            <li><%= link_to "Users", '#' %></li>
+               <li><%= link_to "Users", users_path %></li>
```

Comment:
@johnedelmapa Try to properly indent this section here, inside the `if/else` blocks.